Cargo.toml

```diff
@@ -13,43 +13,41 @@ debug-render = []
 metrics = []
 
 [dependencies]
-bevy_mod_picking = "0.20"
-bevy_vello = { version = "0.5", features = ["svg"] }
-bevy-inspector-egui = "0.25"
-bevy-trait-query = { git = "https://github.com/RobWalt/bevy-trait-query", rev = "215b9068e8457d96ddcd4c8efa9605ba0f8feda5" }
+bevy_vello = { path = "../bevy_vello", features = ["text", "default_font"] } #{ version = "0.8", features = ["svg", "text", "default_font"] }
```

Comment:
TODO: Replace with official bevy vello release. Should maybe come on mondayish(May 5th).